examples/gatsby/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "endOfLine": "lf",
```

Comment:
I don't think it's illustrative to have a formatter in this example, it's orthogonal to what we want to show

Reply:
Whoops that is from the `gatsby new` tool.

Reply:
deleted.